javascript/javascript1/week2/answer.js

```diff
@@ -0,0 +1,146 @@
+//Flight booking fullname function
+/*
+this function returns a string concatenating
+firstname and surname and also checks if the name needs to be addressed formally,
+if so then checks if the person is a woman and returns the string accordingly
+*/
+function getFullname(firstname, surname, useFormalName, isWoman) {
+  if (useFormalName) {
+    if (isWoman) {
+      return "Madam " + firstname + " " + surname;
+    } else {
+      return "Lord " + firstname + " " + surname;
+    }
+  } else {
+    return firstname + " " + surname;
+  }
+}
+
+const fullname1 = "Benjamin";
```

Comment:
I know this is how the variable is written in the exercise requirements, but it would be better to be in camelCase. The same is valid for the function name.